release/scripts/PowerShell/FhirServerRelease/Public/Add-AadTestAuthEnvironment.ps1

```diff
@@ -109,6 +109,15 @@ function Add-AadTestAuthEnvironment {
         $application = Get-AzureAdApplicationByIdentifierUri $fhirServiceAudience
     }
 
+    Write-Host "Getting storage accounts in resource group"
```

Comment:
Is this just used for testing?

Reply:
Yes. These storage accounts are deployed with the PR resource group and then torn down with that environment. We need access to the storage accounts from the test code in order to read the exported data.